include/aws/auth/signing.h

```diff
@@ -82,6 +83,53 @@ int aws_sign_request_aws(
     aws_signing_complete_fn *on_complete,
     void *userdata);
 
+/**
```

Comment:
Normally we shove these sorts of functions into `aws/<lib>/private/` to further indicate their internal-ness. They are still installed, but only referenced in tests.

Reply:
Actually, these are bound out in the CRTs so that SDKs can check their answers too.

Reply:
Oh. Nevermind then!